packages/opentelemetry-tracer-web/test/StackScopeManager.test.ts

```diff
@@ -0,0 +1,175 @@
+/*!
+ * Copyright 2019, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as assert from 'assert';
+import { StackScopeManager } from '../src';
+
+// @ts-ignore
+if (typeof global.window === 'undefined') {
```

Comment:
I would think the `opentelemetry-tracer-web` tests would only ever be meant to run in a browser context, not a Node context.

Given that, could you modify `package.json` to make it so that running the Node tests is just a no-op so we can remove this? As a minor other pro to that approach, the CI build will save a bit of time from not having to run these tests in Node.